config/exercise-readme-insert.md

```diff
@@ -27,4 +27,5 @@ The [exercism/rust](https://github.com/exercism/rust) repository on GitHub is th
 If you want to know more about Exercism, take a look at the [contribution guide](https://github.com/exercism/docs/blob/master/contributing-to-language-tracks/README.md).
 
 [help-page]: http://exercism.io/languages/rust
-[crates-and-modules]: http://doc.rust-lang.org/stable/book/crates-and-modules.html
```

Comment:
this is not on you, but I think this was unused. If it were used, I think I would have seen a link that looks like`[Crates and modules][crates-and-modules]`. We might want to look into that, and maybe a future PR can simply remove the `[modules]:` and `[cargo]:` if they are similarly not used. But I would keep what's been done in this PR, since it is just doing what it says it's doing (replace 1st ed with 2nd ed)

Reply:
Oh, I forgot how named links in markdown work. That makes sense.